docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,80 @@
-memory-mapped input/output (MMIO) base address (MMIOH base) and range (MMIO high size) in the BIOS.
-
-In the Supermicro system in the system bios you need to see the following
-
-  * Advanced->PCIe/PCI/PnP configuration-\> Above 4G Decoding = Enabled
-  * Advanced->PCIe/PCI/PnP Configuration-\>MMIOH Base = 512G
-  * Advanced->PCIe/PCI/PnP Configuration-\>MMIO High Size = 256G
-
-When we support Large Bar Capability there is a Large Bar VBIOS which also disable the IO bar.
-
-For GFX9 and Vega10 which have Physical Address up 44 bit and 48 bit Virtual address.
-
-  * BAR0-1 registers: 64bit, prefetchable, GPU memory. 8GB or 16GB depending on Vega10 SKU. Must
-    be placed < 2^44 to support P2P  	access from other Vega10.
-  * BAR2-3 registers: 64bit, prefetchable, Doorbell. Must be placed \< 2^44 to support P2P access from
-    other Vega10.
-  * BAR4 register: Optional, not a boot device.
-  * BAR5 register: 32bit, non-prefetchable, MMIO. Must be placed \< 4GB.
-
-Here is how our base address register (BAR) works on GFX 8 GPUs with 40 bit Physical Address Limit ::
-
-  11:00.0 Display controller: Advanced Micro Devices, Inc. [AMD/ATI] Fiji [Radeon R9 FURY / NANO
-  Series] (rev c1)
+If your system has a PCIe Express Switch, it must support atomic operations routing. Atomic
+operations requests are permitted only if a component's ``DEVCTL2.ATOMICOP_REQUESTER_ENABLE``
+field is set. These requests can only be serviced if the upstream components also support atomic operation
+completion, can route it to a component that supports it, or both. 
 
-  Subsystem: Advanced Micro Devices, Inc. [AMD/ATI] Device 0b35
+ROCm also uses the PCIe-ID-based ordering technology for peer-to-peer (P2P) data transmission when the GPU
+initiates two write operations to two memory locations. As an example scenario, there are two write operations which needs to be executed in the defined order:
 
-  Flags: bus master, fast devsel, latency 0, IRQ 119
+1. Write to another GPU memory.
+2. Write to system memory, to indicate the transfer is complete.
 
-  Memory at bf40000000 (64-bit, prefetchable) [size=256M]
+The two write operations are routed to different locations. However, use of the ordering technology ensures that the order of the operation is maintained. 
 
-  Memory at bf50000000 (64-bit, prefetchable) [size=2M]
+For more information on changes implemented in PCIe 3.0, see `Overview of Changes to PCI Express 3.0 <https://www.mindshare.com/files/resources/PCIe%203-0.pdf>`_.
 
-  I/O ports at 3000 [size=256]
+CPUs and I/O devices with PCIe atomics support
+------------------------------------------------
+ROCm requires CPUs that support PCIe atomics. Modern CPUs after the release of first generation AMD Zen CPU and Intel™ Haswell support PCIe atomics. Some of the PCIe Endpoints with support beyond AMD Ryzen, AMD EPYC, Intel™ Haswell, or newer CPUs with PCIe 3.0 support are:
```

Comment:
```suggestion
ROCm requires CPUs that support PCIe atomics. 
```

I'm removing this because this list will go stale.

Reply:
Incorporated. Since there is no list of supported CPUs and devices, the CPUs and I/O devices with PCIe atomics support section is removed. The only info left "ROCm requires CPUs that support PCIe atomics. Similarly, all connected I/O devices should also support PCIe atomics for optimum compatibility", has been added to PCIe for atomic operations section.